### PR TITLE
Use path for node delete

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

File: BackofficeBundle/Manager/NodeManager.php [hunk 1]
```diff
@@ -164,19 +164,18 @@ protected function getEditableStatus(NodeInterface $node = null)
...
-        $parentId = null;
         $siteId = $this->contextManager->getCurrentSiteId();
-        $nodeIdDeleted = array();
         foreach ($nodes as $node) {
-            $nodeId = $node->getNodeId();
-            $node->setDeleted(true);
-            $parentId = $nodeId;
-            if (!in_array($nodeId, $nodeIdDeleted)) {
-                $nodeIdDeleted[] = $nodeId;
+            $nodePath = $node->getPath();
```
Comment (itkg-ngilain) [inline BackofficeBundle/Manager/NodeManager.php]: you can put that line after the test as if the node is already deleted, getting the path is useless

**Comment by itkg-nanne** [comment]
done @itkg-ngilain

**Comment by itkg-ngilain** [comment]
Ok with that update, but there's still something to agree in https://github.com/open-orchestra/open-orchestra-model-bundle/pull/422/files#r38841223 to finalize this

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**